document capacity_cpus TFE option (#1849)
hashicorp/terraform-website: content/source/docs/enterprise/system-overview/capacity.html.md

So a 4-core instance with 16 GB of memory could comfortably run 20 Terraform runs, if the runs are allocated the default
512 MB each.

> As of the `v202109-1` TFE release, the maximum number of CPU cores that can be
> allocated to a Terraform run can be adjusted via the `capacity_cpus` Replicated
> configuration option. The configuration will place a hard quota on the number of
> cores that a Terraform operation (and the underlying provider plugin logic) can
> consume. This can be an effective tool to prevent one expensive workspace from
> monopolizing the CPU resources of the host.
> 

Reviewer: Just a suggestion for flow and style - thank you! 

```suggestion
As of the `v202109-1` TFE release, you can use the `capacity_cpus` Replicated configuration option to set the maximum number of CPU cores that can be allocated to a Terraform run. When `capacity_cpus` is set, the configuration places a hard quota on the number of cores that a Terraform operation and underlying provider plugin logic can consume. This can be an effective tool to prevent one expensive workspace from
monopolizing the CPU resources of the host.
```
